pkg/kube/kubehttp.go

```diff
@@ -129,6 +129,10 @@ func (c *KubeHTTP) Delete(manifests []byte, namespace string) ([]byte, error) {
 	return nil, errors.New("not implemented")
 }
 
+func (c *KubeHTTP) DeleteByName(name, kind, namespace string) ([]byte, error) {
+	return nil, errors.New("not implemented")
```

Comment:
We will probably need this implemented at some point. What might that look like? Is it as simple as deleting something from the cluster?

Reply:
It would have to be based on the dynamic interface but, yes... I figured both `Delete` functions could be taken on as a different task. I'll add an issue